src/client/client4.ts

```diff
@@ -1391,7 +1391,7 @@ export default class Client4 {
     };
 
     updateChannelNotifyProps = async (props: any) => {
-        this.trackEvent('api', 'api_users_update_channel_notifcations', {channel_id: props.channel_id});
+        this.trackEvent('api', 'api_users_update_channel_notifications', {channel_id: props.channel_id});
```

Comment:
@jasonblais I'm adding you as a reviewer as I believe you have more context on analytics/tracking events.  TIA.

Reply:
if this is valid, please do file a ticket for mobile